### DISCUSSION TITLE
Cannot see the 5G network when using pixel 7a and USRP B200

### DISCUSSION THREAD

**Discussion by chonghua1** [srsran/srsRAN_Project#542] [General Help] [answered]
I'm using Pixel 7A to test the connection between gNB, which uses USRP B200, but I can't search the 5G network in UE. The output of gNB and CN looks like this:
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/299ccc96-f4f5-441e-b69b-7fbfe001c25c)
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/dfd93ed5-6995-476f-bed0-df82b2058738)

It seems like the gNB successfuly connect with CN, but doesn't transmit the SIB signal.

**Comment by ffxxcad** [comment]
Have you had success in connecting other COTS UEs or an Amarisoft UE?

**Reply by chonghua1** [reply]
haven't try it for now, I can't see the 5G network with current UE, and this is my SIM card configuration:

![image](https://github.com/srsran/srsRAN_Project/assets/42130689/12ab1767-ab64-40e4-aec6-5dafe3d91ff1)

![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=3083770430243726561 skey=@crypt_777de82d_69993ad01172c0b53fb4cac9228c1063 mmweb_appid=wx_webfilehelper](https://github.com/srsran/srsRAN_Project/assets/42130689/5a549e7f-d118-4f32-9894-8ff5d530add7)
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=3083770430243726561 skey=@crypt_777de82d_69993ad01172c0b53fb4cac9228c1063 mmweb_appid=wx_webfilehelper](https://github.com/srsran/srsRAN_Project/assets/42130689/b914f4fe-eb0e-40a0-b29b-212ce3dfc1de)

**Reply by chonghua1** [reply]
![default](https://github.com/srsran/srsRAN_Project/assets/42130689/8b27740f-4c1f-4987-87c1-7687333cf9c5)

**Reply by ffxxcad** [reply]
For your SIM configuration, try changing the `PLMNwAct`, `OPLMNwAct`, and `HPLMNwAct` to `00101:4000` and the `EHPLMN` to `00101`.

Also, have you updated the PLMN to 00101 in the nrf.yaml and amf.yaml for the core?

**Reply by ffxxcad** [reply]
I also recommend you install the [5G Switch App](https://play.google.com/store/apps/details?id=com.sladjan.sava.petg&hl=en_US&gl=US) on the phone and select `NR only` so that the phone only sees 5G Networks when searching.

**Reply by chonghua1** [reply]
Thank you, I wiil try it! 
I have followed the srsRAN tutorial [https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html](https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html), this is the amf.yaml and nrf.yaml configuration:
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/c023abe8-3750-4722-862c-52aa4f3906f3)
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/42eebb26-056d-4b66-b237-374364d711a0)

**Reply by chonghua1** [reply]
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/793dd421-88e9-460a-9db6-48c3832430bc)

**Reply by ffxxcad** [reply, marked as answer]
That tutorial is customized for connecting a Oneplus 8T, which tends to connect better when roaming, thus the PLMN of 90170. However, In my experience the roaming hack is not needed when using Pixel phones.

I would change the amf and nrf configs to match your SIM configuration:
```
mcc: 001
mnc: 01
```

**Reply by chonghua1** [reply]
I have changed the configuration metioned above, but the 5G network still NOT Visible.... I try to use another hackrf to see the spectrum, it seems like no signal there, does this mean the gNB isn't transimitting the SIB broadcast signals properly?
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=2146104979801865793 skey=@crypt_777de82d_69993ad01172c0b53fb4cac9228c1063 mmweb_appid=wx_webfilehelper](https://github.com/srsran/srsRAN_Project/assets/42130689/eb1b1870-63ef-4309-9edb-f6069360810c)

**Reply by pgawlowicz** [reply]
could you share your gnb config?

**Reply by chonghua1** [reply]
Dear @pgawlowicz, thanks for the reply, I followed the suggestions of @ffxxcad and changed my USRP's antenna, and the problem has been solved 😄

**Reply by Zplusless** [reply]
Hi, @chonghua1 , Can you please share how to change the antenna? Thanks!

**Reply by chonghua1** [reply]
Hi @Zplusless, I just changed the antenna to the VERT2450, which was bought from Ettus

**Reply by Zplusless** [reply]
@chonghua1 thanks！
